src/components/tutorial.jsx

```diff
@@ -15,6 +15,7 @@ if (window.tutorialsCompletedThisSession) {
 class Tutorial extends React.Component {
   componentWillMount() {
     this.previousActiveElement = document.activeElement;
+    this.handleNextClick = this.handleNextClick.bind(this);
```

Comment:
I think both of these could go into a class constructor (`componentWillMount` is likely going to be deprecated)

Reply:
Oh neat. I never thought to make a function call in a class constructor.

Reply:
Oh wait. Scratch that. I didn't notice when making this change that this was `componentWillMount` and not the constructor. Not firing on all cylinders today.